includes/amp-helper-functions.php

```diff
@@ -1067,3 +1067,60 @@ function amp_wp_kses_mustache( $markup ) {
 	$amp_mustache_allowed_html_tags = array( 'strong', 'b', 'em', 'i', 'u', 's', 'small', 'mark', 'del', 'ins', 'sup', 'sub' );
 	return wp_kses( $markup, array_fill_keys( $amp_mustache_allowed_html_tags, array() ) );
 }
+
+/**
+ * Add "View AMP" admin bar item for Transitional/Reader mode.
+ *
+ * Note that in Transitional mode, the admin bar item will be further amended by `AMP_Validation_Manager::add_admin_bar_menu_items()`.
```

Comment:
Actually, `AMP_Validation_Manager::add_admin_bar_menu_items()` is responsible for both Transitional _and_ Native modes.